### PR TITLE
Add react-native-elements@0.11.2 w/ npm auto-update

### PR DESCRIPTION
**Pull request by sufuf3**
@kennynaoh please help me review this PR, thank you.

Pull request for issue: #9789 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/react-native-community/react-native-elements
 * Official website (optional, not the repository): https://react-native-training.github.io/react-native-elements/
 * NPM package url (optional): https://www.npmjs.com/package/react-native-elements
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 167
   - Count of stars: 5066
   - Count of forks: 410
 * NPM download stats (optional):
   - Downloads in the last day: 583
   - Downloads in the last week: 3919
   - Downloads in the last month: 14177

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
@kennynaoh please help me review this PR, thank you.

**Comment by sufuf3** [comment]
ping @kennynaoh

**Comment by kennynaoh** [comment]
@sufuf3 Should we need all the files under `src/`?
And there are many minified file show error.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by Monte9** [comment]
@PeterBot @PeterDaveHello Yes! We are actively working on this. Should have it ready within the next week or so. 

You can follow the updates here: https://github.com/react-native-training/react-native-elements/commit/ba3040130356dddfd182fb281b6ce20aa4e8f776#commitcomment-21644096

**Comment by PeterDaveHello** [comment]
@Monte9 sounds great, thank a lot for the updates!

**Comment by sufuf3** [comment]
@Monte9 Thanks, it's great!

**Comment by Monte9** [comment]
Yes, this is ready! Here's the `bundle.js` file for React Native Elements.

https://github.com/react-native-training/react-native-elements/blob/master/build/bundle.js

**Comment by sufuf3** [comment]
@Monte9 Thank you very much.
@kennynaoh @PeterDaveHello should I just add  the `bundle.js` which is minified file after v0.11.0, and exclude the versions before v0.10.0 (via `.gitignore`) ?

And the "basePath" is "dist".

**Comment by PeterDaveHello** [comment]
Let's do it.

**Comment by sufuf3** [comment]
@cdnjs/intern3 please help me review this PR, thanks.
The files under `src/` can't minify, the author provides the file which is minified.
Please refer to the above comments.

File: ajax/libs/react-native-elements/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "react-native-elements",
+  "version": "0.11.2",
+  "description": "React Native Elements & UI Toolkit",
+  "filename": "bundle.js",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/dabit3/React-Native-Elements.git"
```
Comment (extend1994) [inline ajax/libs/react-native-elements/package.json]: I know this is the same as the upstream, just wonder if this is okay?

Comment (sufuf3) [inline ajax/libs/react-native-elements/package.json]: I use `git clone ssh://git@github.com/dabit3/React-Native-Elements.git` and it works.
But I have the question as same with you.

Comment (kennynaoh) [inline ajax/libs/react-native-elements/package.json]: Maybe we can just use `git@github.com/dabit3/React-Native-Elements.git`.

**Comment by sufuf3** [comment]
@kennynaoh We don't build files from src by our own, so we host a single file as `bundle.js`.
There is no `bundle.js` before v0.10.3, so I think these versions can ignore via `.gitignore`. ref: https://github.com/cdnjs/cdnjs/pull/10418#issuecomment-297217470
The `bundle.js` provided after v0.11.0.

**Comment by kennynaoh** [comment]
So please explain it in commit message.

File: ajax/libs/react-native-elements/package.json (hunk 1 above)
Comment (sufuf3) [inline ajax/libs/react-native-elements/package.json]: @PeterDaveHello could you please tell me that should I use the same as the upstream or use `git+https//github.com/dabit3/React-Native-Elements.git`
or 
`git+https://github.com/react-native-training/react-native-elements.git`

Comment (PeterDaveHello) [inline ajax/libs/react-native-elements/package.json]: We don't need to touch it.

Comment (sufuf3) [inline ajax/libs/react-native-elements/package.json]: OK! Thank you.

**Comment by sufuf3** [comment]
@kennynaoh I explained it in the commit message, please help me review this PR, thanks.

**Comment by sufuf3** [comment]
@kennynaoh I remove the `.gitignore` please help me review this PR again, thanks.

**Comment by PeterDaveHello** [comment]
@sufuf3 looks like the file is now non-minified in new releases, can you confirm with upstream?

**Comment by sufuf3** [comment]
OK! I'll discuss with the author.

**Comment by PeterDaveHello** [comment]
@sufuf3 as there is no immediate response, please create an issue for it.

**Comment by sufuf3** [comment]
OK!

**Comment by PeterDaveHello** [comment]
@sufuf3 So have you created it?

**Comment by sufuf3** [comment]
The upstream responded.
https://github.com/react-native-training/react-native-elements/issues/375#issuecomment-302287128
I'll send the issue to add v0.12.0 when I get the answer which that the upstream will provide the minified files in current version (v0.12.0).

**Comment by PeterDaveHello** [comment]
@sufuf3 any related issue in our repo now?

**Comment by PeterDaveHello** [comment]
@sufuf3 should leave the message here earlier next time, also should reference this issue there.

**Pull request merged**